app/models/link_set.rb

```diff
@@ -1,42 +1,26 @@
 class LinkSet < ActiveRecord::Base
-  include Replaceable
-  include DefaultAttributes
-  include SymbolizeJSON
-
-  validate :links_are_valid
+  has_many :links
 
-  def links=(links_hash)
-    super(links_hash || {})
-  end
+  include Replaceable
 
-private
-  def self.query_keys
-    [:content_id]
-  end
+  def hashed_links
```

Comment:
@tuzz https://github.com/alphagov/publishing-api/pull/149 is specifically made to address this comment. 

Once that is merged, we can than move the logic to `LinkSetPresenter#links` without having to change other parts of the app.